src/components/application_manager/test/application_manager_impl_test.cc

```diff
@@ -941,6 +946,49 @@ TEST_F(ApplicationManagerImplTest, UnregisterAnotherAppDuringAudioPassThru) {
   }
 }
 
+TEST_F(ApplicationManagerImplTest,
+       RegisterApplication_TrueIconPath_PathToTheIconWasSet) {
+  file_system::CreateDirectory(kDirectoryName);
```

Comment:
@BSolonenko All these functions return bool. Check returned values before further processing

Reply:
@AByzhynar Fixed in https://github.com/smartdevicelink/sdl_core/pull/2121/commits/d3669007e038ee0921f3d6fa4b3786b723fed263